modules/ossm-rn-deprecated-features.adoc

```diff
@@ -15,6 +15,16 @@ Deprecated functionality is still included in {product-title} and continues to b
 
 Removed functionality no longer exists in the product.
 
+== Deprecated and removed features in {SMProductName} 2.5
```

Comment:
Per our [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings), every heading needs a unique ID, here's one that would work for this heading:
```suggestion
[id="deprecated-removed-features-ossm-2-5"]
== Deprecated and removed features in {SMProductName} 2.5
```

Feel free to choose a different ID wording/pattern that would work better for duplication in future releases.

Reply:
Done